internal/mode/static/handler_test.go

```diff
@@ -446,6 +485,11 @@ var _ = Describe("eventHandler", func() {
 
 		handler.HandleEventBatch(context.Background(), ctlrZap.New(), batch)
 
+		Expect(helpers.Diff(handler.GetLatestConfiguration(), &dataplane.Configuration{Version: 2})).To(BeEmpty())
+
+		_, ok := <-readyChannel
+		Expect(ok).To(BeFalse())
```

Comment:
I think I may be slightly misunderstanding, but I'm pretty sure that `readyChannel` isn't sent to anywhere. The only current use case of `readyChannel` is blocking select statements that get unblocked when `setAsReady` is called and the channel is closed.

Reply:
You're right; we don't write to the channel anywhere. Is there a test that makes sure that we close the ready channel when the Pod is ready?

Reply:
```		
Expect(handler.cfg.healthChecker.readyCheck(nil)).ToNot(Succeed())
handler.HandleEventBatch(context.Background(), ctlrZap.New(), batch)

Expect(helpers.Diff(handler.GetLatestConfiguration(), &dataplane.Configuration{Version: 1})).To(BeEmpty())

_, ok := <-readyChannel
Expect(ok).To(BeFalse())

Expect(handler.cfg.healthChecker.readyCheck(nil)).To(Succeed())
```

There are a couple of these tests in handler_test that test that the ready channel is closed when the Pod is ready. However, this only tests that its closed and doesn't include any test that the channel was actually open. I was trying to figure out a way to do it here but couldn't figure it out, but I think that part of the channel is tested in `job_test.go` where we know the channel is opened correctly because it blocks for a set time correctly. 

Are these checks in `handler_test` sufficient?

Reply:
Sorry, I'm apparently having a hard time reading code today! You are testing that the channel is closed here, and I just misread it. Please disregard 😅. These tests are fine. 

You could use the gomega matcher `Should(BeClosed())` instead of:

```
_, ok := <-readyChannel
Expect(ok).To(BeFalse())
```

to be more descriptive and prevent people like me from misreading lol.

Reply:
Ah ok, I only had it this way because I didn't know of that gomega matcher. I'll make the change as thats a lot simpler for me to understand too!